api/src/pcapi/routes/public/individual_offers/v1/events.py

```diff
@@ -640,3 +641,33 @@ def get_event_categories() -> serialization.GetEventCategoriesResponse:
         for subcategory in subcategories.EVENT_SUBCATEGORIES.values()
     ]
     return serialization.GetEventCategoriesResponse(__root__=event_categories_response)
+
+
+@blueprint.v1_blueprint.route("/events/show_types", methods=["GET"])
+@spectree_serialize(
+    api=blueprint.v1_event_schema,
+    tags=[constants.EVENT_OFFER_INFO_TAG],
+    response_model=serialization.GetShowTypesResponse,
+    resp=SpectreeResponse(
+        **(
+            constants.BASE_CODE_DESCRIPTIONS
+            | {
+                "HTTP_200": (serialization.GetShowTypesResponse, "The show types has been returned"),
+            }
+        )
+    ),
+)
+@api_key_required
+@rate_limiting.api_key_high_rate_limiter()
+def get_show_types() -> serialization.GetShowTypesResponse:
+    """
+    Get all the event categories, their conditional fields, and whether they are required for event creation.
```

Comment:
[détail] trop de copié-collé :)

```suggestion
    Get all show types, bla bla bla
```